tools/scripts/stl-header-whitelist.txt

```diff
@@ -0,0 +1,62 @@
+# QNX platform system headers
+arpa/inet.h
```

Comment:
Those following `.h` files are either posix lib c header or OS headers which are present in most unix operating systems.

Reply:
I adapted the title.